### DISCUSSION TITLE
Event when impersonating

### DISCUSSION THREAD

**Discussion by labomatik** [laravel/nova-issues#4278] [Ideas & Feature Requests]
- Laravel Version: 9.13
- Nova Version: 4.6
- PHP Version: 8.1
- Database Driver & Version: Mysql

### Description:
In the past I was using the package https://github.com/KABBOUCHI/nova-impersonate to handle impersonate, with the new Nova 4 I'm not able to find any event when impersonating, it's particularly useful when you need to clear some cache associated with a user.
Lab404\Impersonate\Events\TakeImpersonation
Lab404\Impersonate\Events\LeaveImpersonation

An event at the start and end should be very good to do some cleanup before and after.

Thx

**Comment by crynobone** [comment]
This has been implemented and will be available on the next release.
